docs/docs/explanations/advanced-configuration.md

```diff
@@ -283,19 +283,32 @@ domain: demo.nebari.dev
 
 ### Continuous integration and continuous deployment
 
-Nebari uses [infrastructure-as-code](https://en.wikipedia.org/wiki/Infrastructure_as_code) to allow developers and users to request changes to the environment via pull requests (PRs) which then get approved by administrators.
-You may configure a CI/CD process to watch for pull-requests or commits on specific branches.
-Currently, CI/CD can be setup for either [GitHub Actions](https://docs.github.com/en/actions) or [GitLab CI](https://docs.gitlab.com/ee/ci/).
+Nebari uses [infrastructure-as-code](https://en.wikipedia.org/wiki/Infrastructure_as_code) to declare a desired state for the deployment.  By using a git repository with CI/CD configured, teams can more quickly modify their deployment, empowering developers and data scientists to request the changes and have them approved by an administrator.
 
+When a `ci_cd` section is configured within your `nebari-config.yaml`, the `nebari render` command will create or update pipeline definition files which describe a CI/CD process to redeploy Nebari when changes are made to a specified branch.  Currently, Nebari can generate CI/CD for [GitHub Actions](https://docs.github.com/en/actions) and [GitLab CI](https://docs.gitlab.com/ee/ci/).
+
+The CI/CD workflow that is best for you will depend on your organization, but the following approach will be appropriate for most situations.
+
+* An admin creates a GitLab or GitHub project, specifies a protected branch (typically `main`) and configures CI/CD variables (see note below) for the target cloud provider
+* The admin then creates a local copy of repository and creates the desired `nebari-config.yaml` including the `ci_cd` section
+* The admin runs `nebari render` for the first time, which in addition to generating IaC in the `stages` repo, will create either a `.github` folder or a `.gitlab-ci.yml` file containing the deploy steps.  *We strongly recommend the administrator review and understand the steps in these CI/CD files before continuing.*
```

Comment:
This... It may not be as is. The intended "scenario" for a first deployment would require the user to follow either the `guided-init` or `init` itself, where the user will be prompted if they would like to use the CICD. 

Same comment regarding the use of `render` x `deploy.` In theory, the most suitable first workflow would be:
- init
- deploy
- attest the basic deployment works as intended
- add modifications as needed (including the CICD if not enabled in the initialization step)
- deploy (here, I could argue that advanced users could use the `render`)
- push (assuming the appropriate repo already exists and the vars were set)

Reply:
I would, instead, remove the first two points there and start with the assumption that the GitLab or GitHub repo already exists and the variables are already set (as in your note), then expand into the valid settings of the `ci_cd` field (maybe move the example from bellow) in the `nebari-config.yaml`, culminating in the 
> Going forward, any user...

Reply:
I'm going to think on this a bit and follow-up with one more commit

Reply:
After more consideration, I think prescribing specific steps here may be more confusing than helpful.  The "easy way" would just be to have Nebari CLI create GitHub repos and CI/CD files for you in a flow where the deploys happen without much oversight/intervention.  However, more advanced production users (like us on our JATIC program) will want a more complex flow whose specific implementation will (a) be driven by specific organization needs, and (b) require a more advanced Nebari user than is the audience for these intro docs anyway.

So I rewrote this section to sound more like general principles than a step-by-step.  How does this sound now?